experimental/CHANGELOG.md

```diff
@@ -39,6 +39,7 @@ All notable changes to experimental packages in this project will be documented
 * feat(instrumentation-xhr): add applyCustomAttributesOnSpan hook #2134 @mhennoch
 * feat(proto): add @opentelemetry/otlp-transformer package with hand-rolled transformation #2746 @dyladan
 * feat(sdk-metrics-base): shutdown and forceflush on MeterProvider #2890 @legendecas
+* feat(sdk-metrics-base): meter identity #2901 @legendecas
```

Comment:
A more descriptive changelog for end users is probably helpful in this case since it's a behavior change

```suggestion
* feat(sdk-metrics-base): return the same meter for identical input to getMeter #2901 @legendecas
```

Reply:
The returned meter object is not actually the same one, but its internal state is identical. This is because we didn't memo the created meter object and return it again in subsequent calls. Maybe we can return an identical meter object, that way people can do stuff like `meter1 === meter2`. I'll do a quick update.

Reply:
whatever you decide to do, the changelog should be meaningfully descriptive for an end user. "identical input to getMeter results in meters with shared internal state" or similar?

Reply:
Thanks for the suggestion! updated.